src/main/java/de/tum/in/www1/artemis/service/dto/UserDTO.java

```diff
@@ -253,4 +256,12 @@ public boolean isInternal() {
     public void setInternal(boolean internal) {
         isInternal = internal;
     }
+
+    public ZonedDateTime isIrisAccepted() {
```

Comment:
This is probably a legitimate complaint from codacy. `is...` should only be used for booleans. `getIrisAccepted` would be better here.